src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,124 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './TodoList.scss';
+import classname from 'classnames';
+import { getTodo } from '../../api/api';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+type Props = {
+  userId: (todo: Todo) => void,
+};
+
+export const TodoList: React.FC<Props> = ({ userId }) => {
+  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState<Todo[] | []>([]);
+  const [optionValue, setOptionValue] = useState('123');
+
+  const filterTodo = [...todos]
+    .filter(todo => todo.title.toLocaleLowerCase()
+      .includes(inputValue.toLocaleLowerCase()));
+
+  useEffect(() => {
+    getTodo()
+      .then((todosFromServer: Todo[]) => setTodos(todosFromServer));
```

Comment:
you have to handle errors, if getTodo returns an error you should catch the error here